run_api_background.py

```
@@ -0,0 +1,82 @@
+#!/usr/bin/env python3
+"""
+Run API server in background for testing
+"""
+
+import subprocess
+import time
+import sys
+import os
+import signal
+
+def start_api():
```

Ruff (0.11.9)

Missing return type annotation for public function `start_api`

(ANN201)